Calculator/FormulaTests/FormulaTests.swift

```diff
@@ -31,10 +31,30 @@ final class FormulaTests: XCTestCase {
         sut.operators.enqueue(.subtract)
         
         // when
-        let result = sut.result()
+        let result = try! sut.result()
         
         // then
         XCTAssertEqual(result, 14)
     }
-
+    
+    func test_피연산자에_0과_연산자에_나누기가_주어지고_result를호출했을때의_에러는_dividingZero와같다() {
+        // given
+        sut = Formula()
+        sut.operands.enqueue(3)
+        sut.operands.enqueue(0)
+        sut.operators.enqueue(.divide)
+        
+        // when
+        let result = Result {
+            try sut.result()
+        }
+        
+        // then
+        switch result {
+        case .success(_):
+            break
```

Comment:
이 테스트에서는 success가 의도하지 않은 결과이기 때문에 
여기는 `XCTFail`을 활용해 테스트를 실패하게 만들어주는 것이 좋다고 생각합니다 💭
```suggestion
            XCTFail("Not Expected Result")
```